src/components/loader/loader.scss

```diff
@@ -47,11 +47,17 @@ $loader-circumference: ($loader-scale - (2 * $stroke-width)) * 3.14159;
 }
 
 :host {
-  @apply hidden;
-}
+  visibility: hidden;
 
-:host([active]) {
-  @apply flex;
+  &:host([hidden]) {
```

Comment:
Please revert these changes. Remember we discussed a separate PR to ensure the `hidden` attribute gets applied to all components (in `includes.scss`).